changelog.md

```diff
@@ -1,5 +1,10 @@
 # Summary of user-visible changes
 
+## 0.9.4 / 2021-05-05
+
+* Fixed a typo in the Lua Primer page
+* Added a reference link to the Lua File System
+
```

Comment:
For the changelog, we typically only include user-facing changes to Fennel itself (the compiler, the cli, etc), since the changelog mainly exists for people to keep track of differences between releases; no need for changelog updates in the docs, since the changelog is, in a way, a part of the docs 😉

Reply:
Sounds good. Will keep in mind for the future. Thanks!